src/components/ProductSize/ProductSize.js

```diff
@@ -2,12 +2,12 @@ import React from "react";
 import classes from "./ProductSize.module.css";
 import Button from "@material-ui/core/Button";
 
-const ProductSize = ({ size, onClick, id }) => {
+const ProductSize = ({ size, onClick, id, active }) => {
   return (
     <Button
       variant="outlined"
       color="secondary"
-      className={classes.button}
+      className={`${id == active && classes.Active}`}
```

Comment:
same here, active should be received from props. It's not clear that your active here is a activeProductSizeId